kani-driver/src/args/mod.rs

```diff
@@ -62,6 +63,58 @@ pub fn print_deprecated(verbosity: &CommonArgs, option: &str, alternative: &str)
 // By default we configure CBMC to use 16 bits to represent the object bits in pointers.
 const DEFAULT_OBJECT_BITS: u32 = 16;
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
You should be able to use strum here instead of implementing FromStr manually: https://docs.rs/strum/latest/strum/derive.EnumString.html

Reply:
Done in https://github.com/model-checking/kani/pull/3649/commits/482ed1130f4f75c1aa90896092758c2db0250301

Thanks for the suggestion.